recipes/darwin-py/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "darwin-py" %}
+{% set version = "0.5.18" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/darwin-py-{{ version }}.tar.gz
+  sha256: 4bdd5d4d6ea9a1e055601f460a6e66ba90298265f749c83fde1bc8c8f0bafa6f
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - darwin=darwin.cli:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - argcomplete
+    - dataclasses
+    - humanize
+    - numpy
+    - pillow
+    - python >=3.6
+    - pyyaml >=5.1
+    - requests
+    - tqdm
+    - upolygon ==0.1.6
```

Comment:
@simedw I've inspected the darwin-py source to try to reduce this list to the actually in-use dependencies, as you're more familiar with this, could you please confirm this looks right?

The torch and detectron2 dependencies are currently left out, as it seems to me those are optional.